setup.py

```diff
@@ -20,6 +20,7 @@
 
 tests_require = [
     'pytest-invenio>=1.3.2',
+    'isort>=4.2.2',
```

Comment:
pytest-invenio provides `isort` already... What was the issue here?

Reply:
I was trying to run "./run-tests" but it kept saying "isort: command not found". After I added this line and running " pip install -e .[all]" it worked fine.

Reply:
Just tested and it worked only running `pip install -e .[all]`.

Reply:
Ok. Closing.